code/modules/admin/admin_verbs.dm

```diff
@@ -1042,3 +1043,41 @@ var/list/admin_verbs_mod = list(
 	message_admins("[key_name_admin(usr)] has stopped all media.", 1)
 
 	stop_all_media()
+
+/client/proc/SendCentcommFax()
+	set	category = "Fun"
+	set name = "Send Fax"
+	set desc = "Sends a fax to all fax machines."
+
+	var/cntcom_msg = input(src, "Please enter a message to send via secure connection. NOTE: BBCode does not work, but HTML tags do! Use <br> for line breaks.", "Outgoing message from Centcomm", "") as message|null
+	if(!cntcom_msg)	return
+
+	var/customname = input(src, "Pick a title for the report", "Title") as text|null
+
+	for(var/obj/machinery/faxmachine/F in allfaxes)
+		if(! (F.stat & (BROKEN|NOPOWER) ) )
+
+			// animate! it's alive!
+			flick("faxreceive", F)

```

Comment:
You should probably make this into a proc because right now this is copy-pasted from the fax machine code.